fdbserver/ClusterController.actor.cpp

```diff
@@ -1530,11 +1530,18 @@ ACTOR Future<Void> statusServer(FutureStream<StatusRequest> requests,
 			// requests
 			last_request_time = now();
 
+			state Optional<StatusReply> faultToleranceRelatedStatus;
```

Comment:
I think you need to clear `faultToleranceRelatedStatus` after the `while` loop below. Otherwise, `faultToleranceRelatedStatus` won't be updated for the next batch.

Reply:
Yes. I missed this part.

Reply:
Addressed.